infinispan/pom.xml

```diff
@@ -22,167 +22,26 @@
+            <version>${project.version}</version>
+        </relocation>
+    </distributionManagement>
 
     <build>
         <plugins>
             <plugin>
-                <groupId>org.bsc.maven</groupId>
-                <artifactId>maven-processor-plugin</artifactId>
+                <artifactId>maven-deploy-plugin</artifactId>
             </plugin>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
+                <artifactId>maven-jar-plugin</artifactId>
                 <configuration>
-                   <forkMode>once</forkMode>
-                    <excludes>
-                        <!-- Skip a long-running test of a prototype class -->
-                        <exclude>**/ClusteredConcurrentTimestampRegionTestCase.java</exclude>
-                    </excludes>
-                     <!-- 
-                          java.net.preferIPv4Stack :
-                             There are problems with multicast and IPv6 on some
-                          OS/JDK combos, so we tell Java to use IPv4. If you
-                          have problems with multicast when running the tests
-                          you can try setting this to 'false', although typically
-                          that won't be helpful.
-                          
-                          jgroups.ping.timeout=500 :
-                             Tell JGroups to only wait a short time for PING 
-                          responses before determining coordinator. Speeds cluster
-                          formation during integration tests. (This is too
-                          low a value for a real system; only use for tests.)
-                          
-                          jgroups.udp.enable_bundling=false :
-                             Disable the JGroups message bundling feature
-                          to speed tests and avoid FLUSH issue
-                           -->
-                    <argLine>-Djgroups.bind_addr=${jgroups.bind_addr} -Dhibernate.test.validatefailureexpected=true -Djava.net.preferIPv4Stack=true -Djgroups.ping.timeout=500 -Djgroups.ping.num_initial_members=1 -Djgroups.udp.enable_bundling=false</argLine>
-                    <skipExec>${skipUnitTests}</skipExec>
-                    <!-- Apache Lucene uses assertions which currently fail on JDK9: -->
-                    <!-- not sure yet how that is going to be resolved, but it's not an OGM problem. -->
-                    <enableAssertions>false</enableAssertions>
-                    <argLine>${additionalRuntimeArgLine}</argLine>
-                    <dependenciesToScan>
-                        <dependency>org.hibernate.ogm:hibernate-ogm-core</dependency>
-                    </dependenciesToScan>
+                    <!-- We need an empty JAR, otherwise the build will fail -->
+                    <skipIfEmpty>false</skipIfEmpty>
```

Comment:
For Validator, I use the pom packaging for relocation:
https://github.com/hibernate/hibernate-validator/blob/master/relocation/engine/pom.xml
so I don't need to produce a jar.